lib/HOW_TO_INSTALL.MD

```diff
@@ -0,0 +1,15 @@
+### Glue support
+
+In order to support federation to AWS Glue catalog these two jars are added as libraries which have to be installed manually like so:
```

Comment:
I am assuming that these libraries will be added to https://github.com/ExpediaGroup/apiary-waggledance-docker at some point?  If so, can you make a note to that effect with a pointer to the repo?

Reply:
I have to check the docker build next if it pulls the released zip then the jars should be bundled up, I got this in my notes already to double check but good callout.

Reply:
yeah that takes in the the RPM which should already have these files packaged.